src/services/error-handler.js

```diff
@@ -10,7 +10,7 @@ exports.catchIfAny = (error, request, response, next) => {
 
     if (!error.status) {
       // NOTICE: Unexpected errors should log an error in the console.
-      logger.error(`Unexpected error: ${message}`, error);
+      logger.error(`Unexpected error: ${message}`);
```

Comment:
Same comment here too. Especially for unexpected errors we need context for debug purpose.

Can you explain how it fixes the issue described in the ticket, I don't really understand at first sight.
<img width="723" alt="Capture_d’écran_2019-11-29_à_08 30 14" src="https://user-images.githubusercontent.com/664261/70030672-fe666080-15a9-11ea-9a35-ac2a522d2255.png">

Thanks!